### PR TITLE
Portland 2023 speakers

### PR DESCRIPTION
**Pull request by plaindocs**
- :heavy_check_mark: all speakers confirmed
- [ ] one missing pic (not a blocker)
- [x] WIP blog post needs work
- [x] More *light* abstract copy edits welcome

----
:books: Documentation preview :books:: https://writethedocs-www--1896.org.readthedocs.build/conf/portland/2023/news/announcing-speakers/

### CODE REVIEW

File: docs/conf/portland/2023/news/announcing-speakers.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+We're delighted to announce that we've finalized our speakers for Write the Docs {{city}} {{year}}! 
+
+Thank you so much to everyone who took the time to send us a proposal, we literally wouldn't have an event without you!
+
+Full speaker line-up
+--------------------
+
+Every year, we look to bring a wide range of voices to the Write the Docs stage. Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.
+We're really excited about the presentations we've got this year – we hope you are too!
```
Comment (thatdocslady) [inline docs/conf/portland/2023/news/announcing-speakers.rst]:
```suggestion
We hope that you'll be as excited as we are about the presentations we've got this year!
```

File: docs/conf/portland/2023/news/announcing-speakers.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+   :template: {{year}}/speakers-simple-list.rst
+
+Head over to the `Speakers page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speakers/>`_ to see the full abstracts.
+We hope there's something there for everyone to enjoy, and also something that will broaden your horizons a bit.
+
+Tickets are going fast!
+-----------------------
+
+If you don't have your ticket yet, now's a great time to snap one up. Conference tickets have sold out for the last several years, and they go faster along with the speaker announcement, so if you're planning to come, don't leave it to the last minute.
+
```
Comment (thatdocslady) [inline docs/conf/portland/2023/news/announcing-speakers.rst:37]:
```suggestion

Please remember to book your travel only after you purchase a conference ticket. With the conference returning to in-person this year, we are working hard to make sure that our attendees have a safe and comfortable experience, so we must maintain capacity limits and guarantee late-registration tickets if you booked your travel already. 
```

Comment (thatdocslady) [reply docs/conf/portland/2023/news/announcing-speakers.rst:37]: I'm not 100% about this phrasing but I figured we should say something about the in-person aspect. Edits welcome!

**Review by thatdocslady — COMMENTED** [review]
Looks great, a few nits on the blog post

File: docs/conf/portland/2023/news/announcing-speakers.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Announcing Portland conference speakers
+=======================================
+
+Our Portland conference is happening in just under 3 months, and we're excited to see everyone again in {{city}}.
+Now is a great time to start booking your tickets and travel,
+and we hope to see you soon.
+
+The biggest announcement today is our speakers and talks for the conference. 
+Thank you so much to everyone who took the time to send us a proposal,
```
Comment (diggerlilly) [inline docs/conf/portland/2023/news/announcing-speakers.rst]: NP: s/proposal,/proposal;

File: docs/conf/portland/2023/news/announcing-speakers.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+we literally wouldn't have an event without you!
+
+Full speaker line-up
+--------------------
+
+Every year, we look to bring a wide range of voices to the Write the Docs stage. Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.
```
Comment (diggerlilly) [inline docs/conf/portland/2023/news/announcing-speakers.rst]: Not crucial, but I stumble over this sentence when reading it... maybe s/look to/try to ?

File: docs/conf/portland/2023/news/announcing-speakers.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+   :template: {{year}}/speakers-simple-list.rst
+
+We hope there's something there for everyone to enjoy, and also something that will broaden your horizons a bit.
+
+Buy your tickets soon
+---------------------
+
+If you don't have your ticket yet, now's a great time to start the process to attend.
+Conference tickets have sold out in the past,
+and tickets sell faster after with the speaker announcement,
```
Comment (diggerlilly) [inline docs/conf/portland/2023/news/announcing-speakers.rst]: ~~with~~

File: docs/conf/portland/2023/news/announcing-speakers.rst [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+so don't leave it to the last minute.
+
+Please remember to book your travel only after you purchase a conference ticket. With the conference returning to in-person this year, we are working hard to make sure that our attendees have a safe and comfortable experience, so we must maintain capacity limits and guarantee late-registration tickets if you booked your travel already.
```
Comment (diggerlilly) [inline docs/conf/portland/2023/news/announcing-speakers.rst]: > we must maintain capacity limits and guarantee late-registration tickets if you booked your travel already

It is unclear to me what we are saying here re: late-reg and travel

File: docs/include/conf/schedule.rst [hunk 7]
```diff
@@ -151,14 +154,14 @@ Social Event
 ~~~~~~~~~~~~
 
 The official Write the Docs social!
-Further details will be announced later, but expect a relaxed atmosphere where you can chat and network with your fellow documentarians, with drinks and snacks provided. 
+Full details will be close to the day of the conference,
```
Comment (diggerlilly) [inline docs/include/conf/schedule.rst]: Suggestion:

`Full details will be announced closer to the conference,`

**Comment by ericholscher** [comment]
@diggerlilly Thanks for the suggestions -- fixed them!

File: docs/conf/portland/2023/news/announcing-speakers.rst [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+This means we have capacity limits and can't guarantee late-registration tickets if you book travel without a conference ticket. 
+
+* Read our `Convince your manager page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/convince-your-manager/>`_ to help make the case for attending.
+* Buy your tickets on our `Tickets page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_.
+
+Call for Volunteers
+-------------------
+
+We're also looking for volunteers to help out at the conference.
+Generally volunteers live within driving distance,
```
Comment (thatdocslady) [inline docs/conf/portland/2023/news/announcing-speakers.rst]:
```suggestion
Generally volunteers live within driving distance to the venue,
```

File: docs/conf/portland/2023/news/announcing-speakers.rst [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+or are just excited about helping out with the conference.
+Volunteering will get you a free ticket to the conference in return for working a couple shifts (averaging around 3 hours each) at the event.
```
Comment (thatdocslady) [inline docs/conf/portland/2023/news/announcing-speakers.rst]:
```suggestion
Volunteering will get you a free ticket to the conference in return for working at least two shifts (averaging around 3 hours each) at the event.
```

File: docs/conf/portland/2023/news/announcing-speakers.rst [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+
+This form will stay open until we have enough volunteers signed up for the event.
+Generally you have until mid March
```
Comment (thatdocslady) [inline docs/conf/portland/2023/news/announcing-speakers.rst]:
```suggestion
The volunteers list usually fills up by mid-March, so don't wait too long to sign up.
```

**Review by thatdocslady — APPROVED** [review]
Last few nits, most of them not blockers so approving pending updates

**Pull request merged**